cetloxblock/cetloxblock/cetloxblock.py

```diff
@@ -0,0 +1,165 @@
+# -*- coding: utf-8 -*-
+""" 
+This XBlock displays a CET LO item.
+The studio view enables the author to select the LO for the current xblock instance.
+"""
+
+import pkg_resources
+#import urllib
+from xblock.core import XBlock
+#from xblock.fields import String, Scope, ScopeBase, UserScope, BlockScope
+from xblock.fields import String, Scope
+from xblock.fragment import Fragment
+
+class CetLoXBlock(XBlock):
+  """ 
+  This XBlock displays a CET LO item.
+  The studio view enables the author to select the LO for the current xblock instance.
+  """
+
+  CET_DOMAIN = u".lab.cet.ac.il"
+  ITEM_URL_TEMPLATE = u"//lo{cetdomain}/player/?document={documentid}&language={language}&sitekey=ebag#options=nobar"
+  TASK_URL_TEMPLATE = u"//lo{cetdomain}/player/xblockplayer.html?task={taskid}&sitekey=ebag"
+  #TASK_URL_TEMPLATE = u"//lo{cetdomain}/player/?task={taskid}&sitekey=ebag#options=nobar"
+
+  #############################################################################
+  # ALL STRINGS ARE UNICODE!
+  # ALL FUNCTION DEFS GET UNICODE PARAMETERS AND RETURN UNICODE VALUES!
+  # KEEP THIS SO FOR YOUR HEALTH AND WELL BEING.
+  #############################################################################

```

Comment:
Is this actually needed if using the correct Python version? I don't see that this has been required for many other XBlocks, and they aren't calling `decode` anywere. I may have misunderstood something, so do clarify.

Reply:
I just followed the [tutorial](http://edx.readthedocs.io/projects/xblock-tutorial/en/latest/getting_started/prereqs.html) and installed python 2.7. Which version do you recommend for me to install?

Reply:
Yep, you're correct, Open edX runs python 2.7.  Just not sure why you need to `decode` all the strings?

Reply:
Only because otherwise I get errors regarding enoding...